quickstarts/maven/spring-boot-with-jib/tempDir/testFile.txt

```diff
@@ -0,0 +1,15 @@
+====
```

Comment:
nit: The name of this file's directory makes me think that it was persisted by accident.
something like 'static/testFile.txt` sounds more suitable/standard.